common/app/views/fragments/containers/facia_cards/container.scala.html

```diff
@@ -79,6 +81,29 @@
                         </div>
                     }
                 }
+
+                @defining((
+                    containerDefinition.index == 0,
+                    frontId.getOrElse(pageId)
+                    )) { case (isFirstContainer, id) =>
+
+                    @if(mvt.ABNewHeaderVariant.isParticipating && isFirstContainer) {
+
+                        <ul class="tertiary-navigation mobile-only">
+
+                            @NewNavigation.TertiaryNav.getTertiaryNavLinks(id).map { footballNavItem =>
```

Comment:
Yes! Thanks for spotting. That should definitely be something more generic. renamed to  `tertiaryLink`